integration-cli/docker_cli_run_unix_test.go

```diff
@@ -1564,7 +1564,8 @@ func (s *DockerSuite) TestRunWithNanoCPUs(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Equals, "50000\n100000")
 
 	out = inspectField(c, "test", "HostConfig.NanoCpus")
-	c.Assert(out, checker.Equals, "5e+08", check.Commentf("setting the Nano CPUs failed"))
+	nanoCpu, _ := strconv.ParseFloat(out, 64)
```

Comment:
which part of our code caused this to fail?

Reply:
@petrosagg I don't know, but it fails on my machine when I run on our branch and doesn't fail on a offical moby. Tested couple of times.

Reply:
@zozo123 ok, can you find out why?